examples/node/scripts/profile/interests.ts

```diff
@@ -0,0 +1,26 @@
+import { LensClient, development } from "@lens-protocol/client";
+
+async function main() {
+  const lensClient = new LensClient({
+    environment: development,
+  });
+
+  // add interests
+  await lensClient.profile.addInterests({
+    interests: ["TECHNOLOGY", "HEALTH_FITNESS__EXERCISE"],
```

Comment:
the interest values should be an enum I guess, how to keep consistency otherwise? there used to be an API endpoint to get all values for interests, but we could also just hardcode them as enum in the sdk.

Reply:
there is the `profileInterestsOptions` endpoint but the interests are returned as strings

shall we create the enum on sdk side or API side?